modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -69,14 +74,15 @@
         <div class="apos-doc-editor__utility">
           <AposSchema
             v-if="docReady"
-            :schema="schemaUtilityFields"
-            :current-fields="utilityFields"
+            :schema="groups['utility'].schema"
+            :current-fields="groups['utility'].fields"
             :trigger-validation="triggerValidation"
             :utility-rail="true"
-            :following-values="followingValues('utility')"
+            :following-values="followingUtils"
```

Comment:
suspect: in the case of editing existing pages, slug's `followingValues` kept triggering their `watch` function running a never ending loop. Could not find anything mutating those values upstream, but capturing the value seemed to stop the run away train

Reply:
Weird. We'll see. I assume slug still follows title.

Reply:
it does in all cases